src/_includes/layouts/post.njk

```diff
@@ -15,7 +15,7 @@ time #}
 {% block content %}
     <main id="site-main" class="post-template site-main outer">
         <div class="inner {{ "ad-layout" if (adsEnabled) }}">
-            <article class="post-full post }}">
+            <article class="post-full post">
```

Comment:
Totally missed these braces before 🤦 

Thanks for catching this and for the fix.